src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuck.java

```diff
@@ -235,7 +237,7 @@ private static void gunzipInPlace(File file) {
     }
 
     private static void applySubsetSingleFileNcks(File file, GoGoDuckModule module) {
-        List<String> ncksSubsetParameters = module.getNcksSubsetParameters().getNcksParameters();
+        List<String> ncksSubsetParameters = module.getNcksSubsetParameters(file.getAbsoluteFile().toString()).getNcksParameters();
```

Comment:
Only need to do this once up front which may mesh better with sourcing from config.  If the files don't have the same lat/lon/time variables we've got no hope of performing an aggregation.